hasjob/static/js/app.js

```diff
@@ -211,12 +211,15 @@ $(function() {
   
   var scrollheight = $('#hgnav').height() - $('#hg-sitenav').height();
   $(window).scroll(function() {
-    if($(window).width() > 767) {
-      if ($(this).scrollTop() > scrollheight) {
-        $('.header-section').slideUp();
-      }
-      else{
-        $('.header-section').slideDown();
+    // Auto-scroll only if a drop-down filter is not open
+    if (!$('.filters .btn-group').hasClass('open')) {
```

Comment:
Doesn't this test slow down the site since it fires on every scroll event? Better if the drop down set a global variable and this test only examines that variable.